src/gui/StyleSpec.h

```diff
@@ -154,6 +157,8 @@ class StyleSpec
 			return STATE_HOVERED;
 		} else if (name == "pressed") {
 			return STATE_PRESSED;
+		} else if (name == "focused") {
+			return STATE_FOCUSED;
```

Comment:
It seems preferable to me to keep these if statements in the same order as the `State` enum.